arm-web/2016-09-01/swagger/AppServicePlans.json

```diff
@@ -3075,47 +3064,36 @@
     "VnetInfo": {
       "description": "Virtual Network information contract.",
       "type": "object",
-      "allOf": [
-        {
-          "$ref": "#/definitions/Resource"
-        }
```

Comment:
Confirming - Is this intentional? removal of "Resource" hence making VnetInfo not an ARM tracked resource anymore?